docs/docusaurus.config.js

```diff
@@ -0,0 +1,87 @@
+module.exports = {
+  title: 'Chart.js',
+  tagline: 'Open source HTML5 Charts for your website',
+  url: 'https://chartjs.org',
+  baseUrl: '/',
+  favicon: 'img/favicon.ico',
+  organizationName: 'chartjs', // Usually your GitHub org/user name.
+  projectName: 'chartjs.github.io', // Usually your repo name.
+  plugins: ['@docusaurus/plugin-google-analytics'],
+  scripts: ['https://www.chartjs.org/dist/next/Chart.min.js'],
+  themeConfig: {
+    googleAnalytics: {
+      trackingID: 'UA-28909194-3',
+      // Optional fields.
+      anonymizeIP: true, // Should IPs be anonymized?
+    },
+    disableDarkMode: true, // Would need to implement for Charts embedded in docs
+    navbar: {
+      title: 'Chart.js',
+      logo: {
+        alt: 'Chart.js Logo',
+        src: 'img/logo.svg',
+      },
+    },
+    footer: {
+      style: 'dark',
+      links: [
+        {
+          title: 'Other Docs',
+          items: [
+            {
+              label: 'Samples',
+              href: 'https://www.chartjs.org/samples/latest/',
```

Comment:
This might not be good for `next`, or is this altered somewhere?

Reply:
I fixed it to have the deploy script update it to point at the right location